frontend/packages/topology/console-extensions.json

```
@@ -1,4 +1,12 @@
 {
   "$schema": "../console-dynamic-plugin-sdk/dist/schema/console-extensions.json",
-  "data": []
+  "data": [
+    {
+      "type": "console.redux-reducer",
+      "properties": {
+        "scope": "devconsole",
+        "reducer": { "$codeRef": "reducer.default" }
```

```suggestion
        "reducer": { "$codeRef": "reducer" }
```

`default` is always the default export name :smiley: